openpype/hosts/blender/blender_addon/addons/UpdatePathsToLastVersion.py

```diff
@@ -81,17 +79,22 @@ class UpdateObjectsPathsVersion(bpy.types.Operator):
 
     def execute(self, context):
         mesh_objects = get_objects_by_type(ObjectsTypes.MESH.value)
-        modifiers_cache_files = [
-            modifier.cache_file for modifier in
-            get_modifiers_by_type(ModifiersTypes.MESH_SEQUENCE_CACHE.value)
-        ]
+        mesh_sequence_caches = get_modifiers_by_type(
+            modifier_type=ModifiersTypes.MESH_SEQUENCE_CACHE.value,
+            given_objects=mesh_objects
+        )
+        modifiers_cache_files = [modifier.cache_file for modifier in mesh_sequence_caches]
         create_library_override_for(
             collections=bpy.context.scene.collection,
             objects=mesh_objects,
             modifiers_cache_files=modifiers_cache_files
         )
+        logging.info('MESH OBJECTS')
+        logging.info(mesh_objects)
+        logging.info([meshe_sequence_cache for meshe_sequence_cache in mesh_sequence_caches])
```

Comment:
Why converting a list to a list?

Reply:
Sorry, it was just for testings purposes. All these logging needs to be deleted.